src/runtime/virtcontainers/utils/utils_test.go

```diff
@@ -151,14 +151,19 @@ func TestCalculateMilliCPUs(t *testing.T) {
 	assert := assert.New(t)
 
 	n := CalculateMilliCPUs(1, 1)
-	expected := uint32(1000)
+	expected := float32(1)
```

Comment:
Ok, I get it now... `CalculateMilliCPUs()` is now returning CPUs in float. It should be renamed to something like  `CalculateCPUsF()` and I'd personally push this semantic change to a separate patch :wink: .